[2024-03-15 16:30:06,347] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_dt_score 2024-03-15T16:29:00+00:00 [queued]>
[2024-03-15 16:30:06,371] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_dt_score 2024-03-15T16:29:00+00:00 [queued]>
[2024-03-15 16:30:06,371] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 16:30:06,371] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-15 16:30:06,371] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 16:30:06,384] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_dt_score> on 2024-03-15T16:29:00+00:00
[2024-03-15 16:30:06,394] {standard_task_runner.py:52} INFO - Started process 22455 to run task
[2024-03-15 16:30:06,397] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_dt_score', '2024-03-15T16:29:00+00:00', '--job-id', '2523', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp8zb_upm3', '--error-file', '/tmp/tmp2ig0e_ua']
[2024-03-15 16:30:06,397] {standard_task_runner.py:77} INFO - Job 2523: Subtask compute_dt_score
[2024-03-15 16:30:06,460] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_dt_score 2024-03-15T16:29:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-15 16:30:06,540] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_dt_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-15T16:29:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-15T16:29:00+00:00
[2024-03-15 16:30:06,720] {python.py:151} INFO - Done. Returned value was: None
[2024-03-15 16:30:06,733] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_dt_score, execution_date=20240315T162900, start_date=20240315T163006, end_date=20240315T163006
[2024-03-15 16:30:06,778] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-15 16:30:06,810] {local_task_job.py:151} INFO - Task exited with return code 0
